src/Surfnet/StepupGateway/GatewayBundle/Saml/Proxy/ProxyStateHandler.php

```diff
@@ -265,7 +265,7 @@ public function isSecondFactorVerified()
         return $this->get('selected_second_factor_verified') === true;
     }
 
-    public function setVerifiedBySsoOn2faCookie(bool $isVerifiedByCookie): self
+    public function setVerifiedBySsoOn2faCookie(?bool $isVerifiedByCookie): self
```

Comment:
Using an unset method would make things clearer then using a nullable parameter.

Reply:
You are absolutely right, changing that :+1: